source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst

```diff
@@ -8,10 +8,9 @@ Robot Preemptive Troubleshooting
     In *FIRST*\ |reg| Robotics Competition, robots take a lot of stress while driving around the field. It is important to make sure that connections are tight, parts are bolted securely in place and that everything is mounted so that a robot bouncing around the field does not break.
 
 Check battery connections
-------------------------------
+-------------------------
 
 .. image:: images/preemptive-troubleshooting/preCheckBatt.png
```

Comment:
It would be absolutely amazing if we could just... replace this picture.

Reply:
Does the image I added in 5100856 work well?